pagan/pagan.py

```diff
@@ -50,12 +50,15 @@ def save(self, path=DEFAULT_OUTPUT_PATH, filename=DEFAULT_FILENAME):
             os.makedirs(path)
 
         # Cut the .png file ending if one was omitted.
-        if filename[-4:] == ".png":
+        if !str.startswith(extensionT, ".")
```

Comment:
where does this trailing uppercase T come frome? and it's not the same as the above exstension, which should have been extension.